code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -198,7 +198,7 @@
 			icon_state = "secborg"
 			animation_length = 28
 			modtype = "Sec"
-			//speed = -1 Secborgs have nerfed tasers now, so the speed boost is not necessary
+			src << "<span class='userdanger'>While you have picked the security module, you still have to follow your laws, NOT Space Law. For Asimov, this means you must follow criminals' orders unless there is a law 1 reason not to.</span>"

```

Comment:
"Security borgs are still bound by Asimov not Space Law, with all laws and the commands of any AI you are synced to coming first. 
While not required to enforce Space Law, you may do so if the act of doing so does not violate a law or command from any synced AI.
if(asimov)
Arresting a non-harmful human for a non-harmful crime and ignoring a request from them to be let go is a violation of ASIMOV laws.